docs/content/reference/values.txt

```diff
@@ -425,8 +425,8 @@
 |gatewayProxies.gatewayProxy.antiAffinity|bool|false|configure anti affinity such that pods are preferably not co-located|
 |gatewayProxies.gatewayProxy.tracing.provider|string|||
 |gatewayProxies.gatewayProxy.tracing.cluster|string|||
-|gatewayProxies.gatewayProxy.rate_limit_settings.max_tokens|int|||
-|gatewayProxies.gatewayProxy.rate_limit_settings.fill_rate|double|||
+|gatewayProxies.gatewayProxy.rateLimitSettings.max_tokens|int|||
```

Comment:
It would be good to put few words explaining what it is.

Reply:
Actually maybe it shoudl be under a different node. Because looking at the values you have impression that we rate limit all requests. But we are just limiting requests to XDS server.